package.json

```diff
@@ -15,8 +15,13 @@
     "lint": "eslint --ext .js,.vue src",
     "lint:fix": "eslint --ext .js,.vue src --fix",
     "android:dev": "node capacitor.config.generator.mjs --dev && npx cap run android",
+    "android:init": "npx cap init",
     "android:open": "vite build && node capacitor.config.generator.mjs && npx cap open android",
     "android:run": "vite build && node capacitor.config.generator.mjs && npx cap run android",
+    "android:sync": "vite build && node capacitor.config.generator.mjs && npx cap sync android",
+    "android:build": "vite build && node capacitor.config.generator.mjs && npx cap build android",
+    "android:release": "vite build && node capacitor.config.generator.mjs && npx cap build android --release",
+    "android:sign": "npx cap open android --release",
```

Comment:
Are there useful all this commands? My opinion is to have only the minimum for a basic user (debug and release). If someone wants more specific commands they can use the native commands, but not populate the package too much. 
If they are useful, then an explanation in the README can be interesting.